assets/css/climatehub.css

```diff
@@ -1,9 +1,41 @@
-/* Example Section */
-.someCoolClassName {
+
+/*FOOTER*/
+
+/*climate hub logo*/
+.chlogo {
+  width: 70%;
+}
+
+/*Social Media Icons*/
+.fa {

```

Comment:
Could you rename this class as well to something that's easily understood.

Reply:
The classes need to be named this because it is linked into the stylesheet and from the source https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css
aka I tried changing it, but it broke
Do you have any suggestions for how to go around this?

Reply:
Nope it's all good, I totally forgot about this, my bad.